data/talks.yml

```diff
@@ -1,4 +1,4 @@
-schedule:
+talks:
```

Comment:
Please consider the fact that these are not just talks but other kinds of events as well and they are all organized around date and time